youtube_dl/extractor/bandcamp.py

```diff
@@ -33,14 +33,17 @@ class BandcampIE(InfoExtractor):
         'info_dict': {
             'id': '1812978515',
             'ext': 'mp3',
-            'title': "youtube-dl  \"'/\\\u00e4\u21ad - youtube-dl test song \"'/\\\u00e4\u21ad",
+            'title': "youtube-dl  \"'/\\\u00e4\u21ad - youtube-dl  \"'/\\\u00e4\u21ad - youtube-dl test song \"'/\\\u00e4\u21ad",
```

Comment:
I think that's the title is incorrectly parsed, i.e. the test case shouldn't be adjusted.

Reply:
OK, that's how trackinfo lists the title, I'll change how it was done to use the main one…

Reply:
(there is no way that test was passing before using the track title from trackinfo, that's why I went for it, but I can fix this, hopefully)